lighthouse-core/report/html/renderer/crc-details-renderer.js

```diff
@@ -182,7 +183,7 @@ class CriticalRequestChainRenderer {
     for (const key of Object.keys(root.tree)) {
       const segment = CriticalRequestChainRenderer.createSegment(root.tree, key,
           root.startTime, root.transferSize);
-      CriticalRequestChainRenderer.buildTree(dom, tmpl, segment, containerEl, details);
+      CriticalRequestChainRenderer.buildTree(dom, tmpl, segment, containerEl, details, detRenderer);
```

Comment:
`det` !? we at least gotta go with `deets` 😆 

is this just coming up on `100` then?